tests/methods/tests.yaml

```diff
@@ -55,11 +55,14 @@ avas:
    short:
```

Comment:
When learning how to use forte to be honest this set-up of avas-3, avas-5 didn't help me figure out what I was looking for. In future as a group we need to figure out a better way to indicate what the test case is before people open the files. Perhaps we should have a short description of each test case in the manual. I know that will be pretty time consuming but I am willing to help with this task

Reply:
I guess it depends on what kind of information should be included in test cases and what should be included in the manual. The DSRG manual contains a complete list of test cases, but the description of each test case is nowhere near complete.

Reply:
This is a good point. I think that for the C20 example you should add more details. For example, it would be more clear to point out that here we select a different plane for each carbon atom, where each plane is defined by the nearest neighbors of each atom.